pageserver/src/storage_sync.rs

```diff
@@ -360,6 +364,19 @@ impl std::fmt::Display for UploadOp {
     }
 }
 
+macro_rules! upload_queue_items_metric {
+    ($remote_timeline_client:expr, $delta:expr) => {{
+        let remote_timeline_client = &$remote_timeline_client;
+        REMOTE_UPLOAD_QUEUE_UNFINISHED_TASKS
+                .get_metric_with_label_values(&[
+                    &remote_timeline_client.tenant_id.to_string(),
+                    &remote_timeline_client.timeline_id.to_string(),
+                ])
+                .unwrap()
+                .add($delta)
+    }}
+}
+
```

Comment:
This could be a regular function, like this:

[upload_queue_items_metric.patch.txt](https://github.com/neondatabase/neon/files/10067399/upload_queue_items_metric.patch.txt)

Reply:
I added a commit with this change to this PR